pennylane/ops/qubit/state_preparation.py

```diff
@@ -204,10 +201,12 @@ class StatePrep(StatePrepBase):
         as :math:`U|0\rangle = |\psi\rangle`
 
     Args:
-        state (array[complex]): the state vector to prepare
+        state (array[complex] or csr_matrix (or whatever the formal type name is for that)): the state vector to prepare
```

Comment:
```suggestion
        state (array[complex] or csr_matrix: the state vector to prepare
```

Although I am a fan of clarity over formality, I am not sure we should be so informal in the official PL documentation :sweat_smile:

Reply:
Applied at https://github.com/PennyLaneAI/pennylane/pull/6863/commits/f0de53444b09e680633fae2bd0f984555abbdf61